### PR TITLE
added custom message

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@HackToHell @alankritapathak  please review

File: custom_message/custom_msgs/package.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<package format="2">
+  <name>custom_msgs</name>
+  <version>1.0.0</version>
+  <description>
+    Custom Metric messages
+  </description>
+  <maintainer email="gautham.manoharan@rapyuta-robotics.com">Gautham Manoharan</maintainer>
+  <license>Commercial</license>
+
+  <author>Gautham Manoharan</author>
```
Comment (alankritapathak) [inline custom_message/custom_msgs/package.xml]: update this info.
Add one nested message as well

**Comment by HackToHell** [comment]
@amitsingh21 how is data being generated for this?

**Comment by ghost** [comment]
> @amitsingh21 how is data being generated for this?

We will be manually publishing the message from command line like this:
rostopic pub -r 10 custom_msgs/OrderMetric '{<>}'

**Review by alankritapathak — APPROVED** [review]

File: custom_message/custom_msgs/package.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>custom_msgs</name>
+  <version>1.0.0</version>
+  <description>
+    Custom Metric messages
+  </description>
+  <maintainer email="kaka@gmail.com">kaka</maintainer>
```
Comment (HackToHell) [inline custom_message/custom_msgs/package.xml]: This will be public, change to a reasonable email id. Same goes for author. Put opensource@rapyuta.io.

**Review by HackToHell — APPROVED** [review]

**Pull request merged**